planck-cljs/src/planck/io.cljs

```diff
@@ -346,12 +346,67 @@
 (defn ^boolean directory?
   "Checks if dir is a directory."
   [dir]
-  (js/PLANCK_IS_DIRECTORY (:path (as-file dir))))
+  (boolean (js/PLANCK_IS_DIRECTORY (:path (as-file dir)))))
 
 (s/fdef directory?
   :args (s/cat :dir (s/or :string string? :file file?))
   :ret boolean?)
 
+(defn exists?
+  "Checks if f exists on disk."
+  [f]
+  (not (nil? (file-attributes f))))
+
+(s/fdef exists?
+  :args (s/cat :f (s/or :string string? :file file?))
+  :ret boolean?)
+
+(defn- path-elements
```

Comment:
@mfikes this could arguably be public.  Thoughts?

Reply:
Yes. Sounds good.

Reply:
Cool beans.  Can you please triple-check the specs for this and `file-name`?  I'm brand new to Clojure specs.